docs/shared-content-source/docs/modules/develop/pages/use-spark-streamlets.adoc

```diff
@@ -118,6 +118,8 @@ In the Spark streamlet deployment model shown above, we can visualize the chain
 In this architecture, the Spark driver runs the Cloudflow-specific logic that connects the streamlet to our managed data streams, at which point the streamlet starts consuming from inlets.
 The streamlet advances through the data streams that are provided on inlets and writes data to outlets.
 
+In case you are using the cloudflow-contrib model of integration, you need to go through some additional steps to complete the deployment of your Spark streamlets. This https://lightbend.github.io/cloudflow-contrib/docs/0.0.4/get-started/spark-native.html[section] on cloudflow-contrib has more details.
```

Comment:
Do we need this comment?

Reply:
Just thought of adding this here since there is no mention of cloudflow-contrib in this section on spark streamlets.